android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -101,6 +100,22 @@ constructor(
     configurationRegistry.retrieveConfiguration(ConfigType.Navigation)
   }
 
+  fun retrieveRemoteIconsBitmap() {
+    navigationConfiguration.clientRegisters
+      .filterNot { it.menuIconConfig == null }
+      .filter { it.menuIconConfig!!.type == "remote" }
```

Comment:
The string "remote" can be moved to a constant. I think the two filters can also be combined into one.

Reply:
Agree